src/components/HTMLLoader.jsx

```diff
@@ -21,6 +21,7 @@ function HTMLLoader({
 
   useEffect(() => {
     let promise = Promise.resolve(); // Used to hold chain of typesetting calls
+
     function typeset(code) {
       promise = promise.then(() => window.MathJax?.typesetPromise(code()))
```

Comment:
you also need to add an additional check here because `typesetPromise` is not a function error occurring here. 
promise = promise
.then(() =>
if(typeof window?.MathJax !== 'undefined') {
     window.MathJax?.typesetPromise(code())
)}
.catch((err) => logError(`Typeset failed: ${err.message}`));